### DISCUSSION TITLE
Furnace Tracker

### DISCUSSION THREAD

**Discussion by aquifolly** [catppuccin/catppuccin#2010] [Port Requests]
### Will you make the port?

- [ ] Yes, I'm willing to make the port

### How can the suggested port be themed?

it has a colour scheme section at the bottom of the appearance settings
![image](https://user-images.githubusercontent.com/35699052/230873790-4be7a89f-1de4-475b-9632-2d181a155872.png)

### Any additional comments?

https://github.com/tildearrow/furnace

### Submission Guidelines

- [X] I have read and followed Catppuccin's submission guidelines.

**Comment by vanillyn** [comment]
ive actually never seen a catppuccin music tracker
id love it!
ive never really used furnace tracker, but ill try to see what its like

**Comment by ghost** [comment]
Yup, I am fully aware of furnace tracker (and have it installed.) I think I am going to begin work on it tomorrow morning. Considering I am trying to get into using music creation, I'd love to have this theme available so why not?

I'll have to make several versions to accommodate each theme so it'll take a while if I do.

**Reply by vanillyn** [reply]
if it's just a text file, you can just export it then use [puccinier](https://github.com/catppuccin/toolbox#%EF%B8%8F-puccinier) to make it way easier

**Reply by aquifolly** [reply]
it doesn't seem to use hex values

**Reply by aquifolly** [reply]
wait actually, it does, but as signed integers, weird

**Reply by ghost** [reply]
@pinkcreeper100 I mean... 0-255 is kinda unsigned.

**Reply by ghost** [reply]
> if it's just a text file, you can just export it then use [puccinier](https://github.com/catppuccin/toolbox#%EF%B8%8F-puccinier) to make it way easier

I'll look into this. Chances are I can't do a 1:1 export, but it may be close enough.

**Reply by aquifolly** [reply]
It was actually signed integers
